pallets/credit/src/lib.rs

```diff
@@ -263,7 +263,9 @@ pub mod pallet {
         CreditDataAdded(T::AccountId, CreditData, T::BlockNumber),
         CreditDataUpdated(T::AccountId, CreditData, T::BlockNumber),
         CreditScoreSlashed(T::AccountId, u64, T::BlockNumber),
-        GetRewardResult(T::AccountId, EraIndex, EraIndex, u8), //status: 0,Normal; 1-6, Error
+        CreditDataAddedByTraffic(T::AccountId, u64, T::BlockNumber),
```

Comment:
I think it's no need to add block number to this event, substrate already know which block this event belongs to
![image](https://user-images.githubusercontent.com/4054836/147721555-b47e5032-c4d3-4248-9335-3e1eb722c7ef.png)

for example, the `DelegatorReward` doesn't contains a block number field, but deeperscan knows it belongs to 955869